packages/nocodb/package.json

```diff
@@ -122,22 +125,22 @@
     "mkdirp": "^2.1.3",
     "morgan": "^1.10.0",
     "mssql": "^6.2.0",
-    "multer": "^1.4.4",
+    "multer": "^1.4.5-lts.1",
     "mysql2": "^3.2.0",
     "nanoid": "^3.1.20",
-    "nc-help": "0.2.92",
+    "nc-help": "^0.2.88",
```

Comment:
Always use exact version.

Reply:
Updated. Later on we need to use `0.2.93` instead.